README.md

```diff
@@ -25,6 +25,11 @@ It will purge the pull zone if "true" provided. pullZoneId and accessKey inputs
 
 The source directory that should be uploaded.
 
+### `destination`
+
+The destination directory that should be uploaded too in the bunny storage zone. (Example: www).
+The destination should not end with `/` as in `www/`.
```

Comment:
Can you add an example of nested directory as well?

Reply:
Sure, just added an example. Is this what you mean?

Reply:
What I meant was for example "dist/release1/draft". Do we support this? Will it create all the 3 directories?